javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,140 @@
+// Flight booking fullname function
+
+function getFullname(firstname, surname, useFormalName, gender) {
+  if (useFormalName) {
+    switch (gender) {
+      case "male":
+        return "Lord " + firstname + " " + surname;
+        break;
+      case "female":
+        return "Lady " + firstname + " " + surname;
+        break;
+      default:
+        return firstname + " " + surname;
+    }
+  }
+  return firstname + " " + surname;
+}
+
+const fullname1 = getFullname("Benjamin", "Hughes", true, "male");
+const fullname2 = getFullname("Line", "Hansen", true, "female");
+
+console.log(fullname1 + "\n" + fullname2);
+
```

Comment:
this looks good, but you forgot a condition>
'Remember to consider someone calling the function with an empty string as firstname and lastname' 
you could e.g. add a warning?

Reply:
@vulk29 thanks, ofcourse I will add this condition))

Reply:
nice!! :)